src/main/blackbox/blackbox.c

```diff
@@ -812,11 +812,7 @@ static bool sendFieldDefinition(const char * const *headerNames, unsigned int he
             return false; //Someone probably called us again after we had already completed transmission
         }
 
-        charsWritten = blackboxPrint("H Field ");
-        charsWritten += blackboxPrint(headerNames[xmitState.headerIndex]);
-        blackboxWrite(':');
-        charsWritten++;
-
+        charsWritten = blackboxPrintf("H Field %s:", headerNames[xmitState.headerIndex]);
```

Comment:
It's a shame to no longer call blackboxPrint() here, because when writing to flash blackboxPrint() can write the text as one unit instead of making many distinct calls to copy into the flashfs buffer.